config/routes/administrators.rb

```diff
@@ -3,5 +3,6 @@
 }
 
 namespace :administrators do
+  resources :administrator, only: %i[index new edit create update destroy]
```

Comment:
```suggestion
  resources :administrators, only: %i[index new edit create update destroy]
```
ここは複数形ではないか
resourcesの場合は複数形にするのが慣習

Reply:
コントローラーの名前問題で`s`つけるかつけないかで混乱してたっぽいですね。。